### PR TITLE
ants: add version 2.3.5

### PR DESCRIPTION
**Pull request by bcdarwin**
- enabled option to build with MINC support (default True)
- changed homepage url to https://

### CODE REVIEW

File: var/spack/repos/builtin/packages/ants/package.py [hunk 1]
```diff
@@ -15,13 +15,26 @@ class Ants(CMakePackage):
        processing library to which ANTs developers contribute.
     """
 
-    homepage = "http://stnava.github.io/ANTs/"
+    homepage = "https://stnava.github.io/ANTs/"
     url      = "https://github.com/ANTsX/ANTs/archive/v2.2.0.tar.gz"
 
     version('2.2.0', sha256='62f8f9ae141cb45025f4bb59277c053acf658d4a3ba868c9e0f609af72e66b4a')
+    version('2.3.5', sha256='2fddfd5f274a47f1c383e734a7e763b627c4a8383d2d3b9971561f335016bb0a')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/ants/package.py:21]: Can you please move the newest version above v2.2.0?

File: var/spack/repos/builtin/packages/ants/package.py [hunk 2]
```diff
@@ -15,13 +15,26 @@ class Ants(CMakePackage):
...
 
     depends_on('zlib', type='link')
 
+    variant('MINC', values=bool, default=True,
+            description='Build ITK with MINC support')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/ants/package.py]: We usually use lowercase for variant names and variants are Boolean by default:
```suggestion
    variant('minc', default=True, description='Build ITK with MINC support')
```

File: var/spack/repos/builtin/packages/ants/package.py [hunk 3]
```diff
@@ -15,13 +15,26 @@ class Ants(CMakePackage):
...
+
+    def cmake_args(self):
+        args = []
+        minc_support = bool(self.spec.variants['MINC'].value)
+        if minc_support:
+            args.extend([
+                '-DITK_BUILD_MINC_SUPPORT=ON',
+            ])
+        return args
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/ants/package.py]: We have a convenience method in `CMakePackage` to deal with define set based on a variant value:
```suggestion
        return [
            self.define_from_variant('ITK_BUILD_MINC_SUPPORT', 'minc')
        ]
```

**Review by alalazo — CHANGES_REQUESTED** [review]
Thanks for the contribution! A few minor changes requested.

File: var/spack/repos/builtin/packages/ants/package.py (hunk 3 above)
Comment (bcdarwin) [reply var/spack/repos/builtin/packages/ants/package.py]: All fixed, thanks.

**Review by alalazo — APPROVED** [review]

**Pull request merged**